### PR TITLE
Add special glasses for Captain

### PR DESCRIPTION
**Pull request by Scruq445**
Added the Captain's Glasses to their locker, which has Security HUD, Flash Immunity, and Welding Protection so the captain doesn't have to get a pair of security glasses or have to use the security HUD.

### CODE REVIEW

**Comment by muburu** [comment]
hasn't it been discussed several times that captain having sechud just gives them more of a reason to validhunt

**Comment by Scruq445** [comment]
I personally use the HUD to quickly get info on a person at a glance, the captain should just call out the wanted person instead of arresting them themselves, because that's what the entire department is for. Including it's not like the captain can't just get a pair of sec glasses along with their tracker implant at roundstart. It would just make a captains life easier, including if you have a full team of secoffs who already took all the security glasses.

**Comment by Scruq445** [comment]
including the HUD would allow the captain to have people to be cautious of if they are wanted. instead of having someone tell the captain to meet at one place then suddenly crit them.

**Comment by thebadman4662** [comment]
I think captain should also have medhud and diagnostic hud so they can guide medical to patients(and totally not use it for combat) and engineering/sci to damaged borgs. :trollface: 

> so the captain doesn't have to get a pair of security glasses or have to use the security HUD.

Good thing captain does not need to get them in first place. Its not their job after all unless sec is so understaffed they need to do Capcurity.

**Comment by TheDoctor1977** [comment]
This just feels like textbook powercreep, I can understand cap having sec glasses but there’s absolutely no reason for them to have weld protection

**Comment by Cojoke-dot** [comment]
This seems like something the captain does not need. You're giving the captain the best pair of glasses in the game... and you can craft them. Not even the nukie agent has a pair of glasses this good.

File: Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml [hunk 1]
```diff
@@ -235,3 +235,24 @@
   - type: Clothing
     sprite: Clothing/Eyes/Glasses/ninjavisor.rsi
   - type: FlashImmunity
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons]
```
Comment (Cojoke-dot) [inline Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml:240]: You want
[ClothingEyesBase, ShowSecurityIcons, ShowMedicalIcons]

I would also say remove the Construction component, you don't want the crew to craft these

**Comment by Scruq445** [comment]
I understand, maybe I was being alittle too powercreep with it. Clearly this is an item the captain does not need. Closing the PR now

**Pull request closed without merging**